install_config/configuring_authentication.adoc

```diff
@@ -1004,6 +1004,10 @@ oauthConfig:
 ----
 <1> This provider name is prefixed to the GitLab numeric user ID to form an
 identity name. It is also used to build the callback URL.
+<2> When *true*, unauthenticated token requests from non-web clients (like
+the CLI) are sent a `WWW-Authenticate` challenge header for this provider.
+This uses the http://doc.gitlab.com/ce/api/oauth2.html#resource-owner-password-credentials[Resource Owner Password Credentials]
+grant flow to obtain an access token from GitLab. 
 <2> *GitLabIdentityProvider* cannot be used to send `WWW-Authenticate`
```

Comment:
@liggitt Sorry for bothering you, but I found out that this commits added two callouts with the same number. I mean there was `<2>` already and another one was added. I can provide a fix for this if you told me how (merge, remove previous or something else). Thanks.